### PR TITLE
Responsiveness changes

### PR DESCRIPTION
**Pull request by 0xAryanShah**
## What it solves
######Notification popup has to be adjusted		
		Use Mobile App Popup Make Green text on black.
		New Transaction popup : Contract Interaction button, Logos can be made black
		Queued transactions will appear here : text smaller even the SVG
		Transactions/History : Filter Text and Whole Filter Popup[Remove it for mobile]		
		Address Book-> Create Entry : The Box has to fit in the page.
		Address Book-> Import Boxes has to fit in.
		Address Book-> Export Boxes has to fit in
		Settings Contract Version popup opening gnosis git contracts
		Remove safe: box has to fit in.
		New Owner : box has to fit in.
		Notification Icon in Center
		Assets: Send and receive buttons are not seen, format the table too.
		Address Book : The Edit/Da1ete/Send button is not seen.
		Settings -> Owners Edit owner/replace owner is not seen. and formatting of the table.
		Assets: No NFTs Available text size
		Address Book: Export Import alignment is off.
		
# Not Done
[Transactions/History : the text can be in line to look continuous]	
		Address Book : Table has to be adjusted
		The Back UX or opening the left panel is not happening if interacted with the page.
		The button are visible but styling them.

### CODE REVIEW

**Comment by zjesko** [comment]
Lets make another PR for the remaining tasks, that way we can atleast close this. ill add an issue on linear

@SiddarthVijay you can review the PR

File: src/components/Modal/index.tsx [hunk 1]
```diff
@@ -170,7 +170,8 @@ const FooterSection = styled.div<{ withoutBorder: boolean }>`
   display: flex;
   justify-content: center;
   align-items: center;
-  /* border-top: ${({ withoutBorder }) => (withoutBorder ? '0' : '2px')} solid ${({ theme }) => theme.colors.separator}; */
+  /* border-top: ${({ withoutBorder }) => (withoutBorder ? '0' : '2px')} solid ${({ theme }) =>
```
Comment (0xtemujin) [inline src/components/Modal/index.tsx]: Can remove comments

File: src/routes/safe/components/AddressBook/index.tsx [hunk 2]
```diff
@@ -210,10 +206,10 @@ const AddressBookTable = (): ReactElement => {
               color="primary"
               iconType="add"
               iconSize="sm"
-              textSize="xl"
-              style={{textDecoration: 'none'}}
+              textSize={below800 ? 'md' : 'xl'}
+              style={{ textDecoration: 'none' }}
             >
-              Create entry
+              Entry
```
Comment (0xtemujin) [inline src/routes/safe/components/AddressBook/index.tsx:204]: Leave this as 'Create Entry'

File: src/routes/safe/components/Transactions/TxList/styled.tsx [hunk 3]
```diff
@@ -196,7 +195,7 @@ export const StyledTransaction = styled.div`
   grid-template-columns: ${Object.values(gridColumns).join(' ')};
   width: 100%;
 
-   /* &:hover {
+  /* &:hover {
```
Comment (0xtemujin) [inline src/routes/safe/components/Transactions/TxList/styled.tsx]: Remove comments

**Review by 0xtemujin — CHANGES_REQUESTED** [review]

**Review by 0xtemujin — APPROVED** [review]

**Pull request merged**